Tools/autotest/arducopter.py

```diff
@@ -1669,9 +1669,12 @@ def fly_optical_flow_limits(self):
         if ex is not None:
             raise ex
 
-    # fly_autotune - autotune the virtual vehicle
     def fly_autotune(self):
+        """Test autotune mode"""
 
+        rlld = self.get_parameter("ATC_RAT_RLL_D")
```

Comment:
One neat optimisation might have been to throw these into a hash - so you just have a list of things you want to be the same before/after and iterate that list to get the values into a hash.